england/tyne_and_wear_metropolitan_county.js

```diff
@@ -0,0 +1,49 @@
+                        "North Tyneside" : "m 692.86,101.56 c 34.84,-1.16 48.61,-44.39 68.36,0.32 23.26,1.84 56.79,-11.27 56.16,23.07 33.32,-12.23 61.04,-28.21 37.38,-66.4 35.27,-6.13 47.69,-44.47 84.91,-44.34 -8.09,36.87 30.07,-8.75 16.95,40.44 -14.69,16.08 -0.05,59.14 25.05,87.54 40.1,24.37 17.51,39.38 30.34,64.53 22.5,29.65 45.47,66.27 -5.25,62.57 -44.92,22.83 -29,121.55 -99.4,103.06 -55.78,-5.28 -127.61,7.75 -167.04,13.13 11.24,-48.12 -66.77,-9.53 -48.25,-54.05 19.21,-52.76 -42.09,-13.14 -51.69,-49.06 -56.06,27.31 -53.29,-50.27 -13.31,-66.97 -1.13,-46.88 -38.31,-70.84 -83.27,-54.23 -27.92,-5.8 -55.34,-13.44 -23.5,-40.89 -28.81,-50.77 0.4,-81.01 53.51,-59.47 41.36,-3.87 73.57,4.75 90.7,37.48 9.1,3.5 18.86,2.52 28.36,3.26 z",
+                        "South Tyneside" : "m 1140.85,551.15 c -5.3,8.04 -4.46,41.29 -44.17,18.77 -46.94,-23.01 -45.5,51.33 -109.61,23.37 -52.05,-13.77 -96.9,31.59 -151.02,6.73 -46.55,5.27 -42.44,-78.98 -41.46,-109.88 -50.1,15.43 -57.94,-89.48 -14.7,-111.13 47.18,-10.68 97.49,-6.63 145.37,-4.33 60.22,-2.62 31.46,-125.73 112.21,-76.32 13.95,-6.56 24.94,71.12 64.03,70.22 28.59,16.06 20.3,56.52 62.79,67.64 24.81,36 30.79,95.55 0.8,114.69 2.91,-2.63 -24.03,-4.41 -24.24,0.26 z",
+                        "City of Sunderland" : "m 850.29,1128.72 c 14.18,-28.9 33.41,-44.75 1.14,-66.34 9.85,-32.2 -41.44,-48.56 -25.56,-89.25 -11.06,-44.32 -17.33,-90.43 -12.06,-134.06 -33.36,-50.55 -93.94,13.88 -125.04,-24.68 15.88,-51.91 6.68,-107.08 -0.04,-152.5 -45.23,-61.06 51.13,-62.55 91.24,-68.78 47.91,7.21 110.04,26.01 157.52,-0.93 45.3,-10.18 87.29,32.71 116.82,-16.9 31.97,-22.7 78.14,17.96 86.09,-25 22.32,-2.59 44.06,6.66 13.71,18.5 6.87,36.53 4.06,78.2 19.92,101.97 -8.11,1.3 31.74,38.49 -6.54,52.98 2.39,48.82 8.81,98.02 31.62,141.47 -3.88,39.68 -73.59,22.06 -83.16,69.79 -40.1,9.5 -106.63,8.1 -89.18,63.24 27.83,39.15 -6.82,91.93 -23,123.93 -45.95,3.52 -96.43,4.48 -146.85,8.63 l -6.63,-2.08 0,0 z"
```

Comment:
Hello @neveldo,
Thankyou for your query. It has made me think of a rationale. I have added a README, which contains the rationale. Briefly,
- In France, Haute Garonne is not Toulouse. In England, The Borough of Wolverhampton is the City of Wolverhampton. So the ids are names.
- All ids are now short, for good computing. Only exception is 'City of London',. For many people, 'London' means 'Greater London'. And 'City of London', or 'The City', are used to talk of the center.
In the latest commits, all files have been renamed. Also, the SVGs contain many fixes to remove tiny boundary holes.